cachi2/core/utils.py

```diff
@@ -100,15 +174,11 @@ def _copy_using(copy_function: Callable) -> None:
             ignore=shutil.ignore_patterns(destination.name),
         )
 
-    if reflink.supported_at(origin):
-        try:
-            log.debug(f"Copying {origin} to {destination} using reflinks.")
-            _copy_using(reflink.reflink)
-        except reflink.ReflinkImpossibleError:
-            log.debug("Reflink copy failed, falling back to standard copy.")
-            _copy_using(shutil.copy2)
-    else:
-        log.debug(f"Copying {origin} to {destination} using a standard copy.")
+    try:
+        log.debug(f"Copying {origin} to {destination} using fast in-kernel copy.")
```

Comment:
Do we have an issue regarding converting the log format to a lazy variant?
I suppose that was the final decision in https://github.com/containerbuildsystem/cachi2/pull/575

Reply:
We don't have one, I'll create one and fix this instance.